pycode/memilio-simulation/CMakeLists.txt

```diff
@@ -45,7 +54,14 @@ function(add_pymio_module target_name)
     pybind11_add_module(${target_name} MODULE ${PYBIND11_MODULE_SOURCES})
     target_link_libraries(${target_name} PRIVATE ${PYBIND11_MODULE_LINKED_LIBRARIES})
     target_include_directories(${target_name} PRIVATE memilio/simulation/bindings)
-    install(TARGETS ${target_name} LIBRARY DESTINATION memilio/simulation)
+    set_target_properties(${target_name} PROPERTIES
+        INSTALL_RPATH "$ORIGIN;$ORIGIN/lib"
+    )
+    install(TARGETS ${target_name}
+        LIBRARY DESTINATION .
+        ARCHIVE DESTINATION .
+        RUNTIME DESTINATION .
+    )
```

Comment:
Maybe add `# make sure to install shared and static libraries (LIBRARY and ARCHIVE) for the executables (RUNTIME)` or sth similar.
Why the change from memilio/simulation to .? The former seems to be the safer option, no?